implicit/recommender_base.py

```diff
@@ -13,7 +13,7 @@ def fit(self, item_users):
         pass
 
     @abstractmethod
-    def recommend(self, userid, user_items, N=10):
+    def recommend(self, userid, user_items, N=10, filter_items=[]):
```

Comment:
I have decided to use lists, I think that function signatures are nicer and easier to comprehend by the final user.